scripts/findAndMatch.js

```diff
@@ -0,0 +1,319 @@
+//GAME - Find & Match
+
+//To be continued: Match the artist with his/her instrument
+
+var msg1 = document.getElementById("msg1");
+var msg2 = document.getElementById("msg2");
+var msg3 = document.getElementById("msg3");
+var startBtn = document.getElementById("start");
+var stopBtn = document.getElementById("stop");
+var outcome = document.getElementById("outcome");
+var p = document.createElement("h5");
+
+startBtn.onclick = startGame;
+stopBtn.onclick = stopCounter;
+
+var randomNames = [];
+var names = [
+  "Hetfield",
+  "Malmsteen",
+  "Trujillo",
+  "Hammet",
+  "Lars",
+  "Halford",
+  "Heafy",
+  "Lemmy",
+  "Ozzy"
+];
+
+var timeLeft = 5;
+var timer;
+var totalLives = 3;
+var remainingLives = 3;
+var key;
+
+function startCounter() {
+  timer = window.setInterval(counter, 1000);
+}
+
+function counter() {
+  timeLeft--;
+  if (timeLeft < 0) {
+    window.clearInterval(timer);
+    remainingLives--;
+    timeLeft = 5;
+    startCounter();
+  }
+  msg2.innerHTML = timeLeft + " sec remaining";
+  msg3.innerHTML =
+    "<hr>" +
+    remainingLives +
+    " / " +
+    totalLives +
+    " " +
+    '<i class="fas fa-heart" style = "color: indianred"></i>';
+  if (remainingLives == 0) {
+    msg1.innerHTML =
+      "GAME OVER!!!" + "<br><br>" + "You did not guess all the artists...";
+    msg1.style.cssText = "color: indianred; font-size: 28px;";
+    msg2.innerHTML = "";
+    msg3.innerHTML = "Please refresh and try again.";
+    outcome.style.cssText = "display: none;";
+    window.clearInterval(timer);
+    startBtn.disabled = true;
+    stopBtn.disabled = true;
+  }
+}
+
+function stopCounter() {
+  window.clearInterval(timer);
+  startBtn.disabled = false;
+}
+
+function startGame() {
+  timeLeft = 10;
+  remainingLives = 3;
+  randomNames = names.slice();
+  randomNames.sort(function() {
+    return 0.5 - Math.random();
+  }); //randomize an array (BEST WAY)
+  msg2.innerHTML = timeLeft + " sec remaining";
+  msg3.innerHTML +=
+    "<hr>" +
+    remainingLives +
+    " / " +
+    totalLives +
+    " " +
+    '<i class="fas fa-heart" style = "color: indianred"></i>';
+  document.getElementById("instructions").style.cssText = "display: none;";
+  msg1.style.cssText = "color: black;";
+  startBtn.disabled = true;
+  findArtist();
+  buildBoard();
+  startCounter();
+}
+
+function findArtist() {
+  key = randomNames[Math.floor(Math.random() * (randomNames.length - 1))];
+  msg1.innerHTML = "Find " + '<span id = "name">' + key + "</span>";
+}
+
+function buildBoard() {

```

Comment:
In all of your functions you are modifying global variables in your functions. This is GENERALY (there are exceptions) considered a bad practice. Try to think of your functions like they are a shredding machine or smth. Something goes into them via the parameters and based on those parameters something comes out. It's a good practive to have pure functions. Meaning that if you call a function with the same parameters it always returns the same result.

```
function findArtist(randomNames) {
     return randomNames[Math.floor(Math.random() * (randomNames.length - 1))]
}
msg1.innerHTML = "Find " + '<span id = "name">' + findArtist(randomNames) + "</span>";
```

This way you will be able to reuse your function in other places. It doesn't need to have randomNames in it's current scope/context, you pass it to it explicitly using the parameters.

It goes kinda the same way for most of the other functions, except the ones that are manipulating the DOM directly, you can't do anything about those. It's generally a good practice to have your computation separated from DOM manipulation / rendering.